content/gherkin/gherkin-reference.md

```diff
@@ -49,30 +49,44 @@ There are a few extra keywords as well:
 ## Feature
 
 A `.feature` file is supposed to describe a single Feature of the system, or a
-particular aspect of a Feature. It's just a way to provide a high-level description
+particular aspect of a Feature. It's a way to provide a high-level description
 of a software Feature, and to group related Scenarios.
 
-A Feature has three basic elements---the `Feature:` keyword, a *name* (on the same line)
-and an optional (but highly recommended) *description*, which can span multiple lines.
+The `.feature` file starts with the keyword **Feature**, a *name* (on the same line),
+followed by an optional (but highly recommended) free format *description* which can span multiple lines.
+The free format description ends when the first scenario starts.
 
-Cucumber does not care about the name or the description. Their purpose is simply to provide a place for you to document important aspects of the Feature, such as a brief explanation and a list of business rules (general acceptance criteria).
+Cucumber does not care about the name or the description.
```

Comment:
Slightly wary of "does not care" here. Can we reword?

Also remove "simply" ?

Reply:
Changed to: "The name and the description have no special meaning to Cucumber." (and removed "simply").
Let me know if anyone has a better idea